inference-engine/ie_bridges/python/tests/test_InferRequest.py

```diff
@@ -452,3 +452,64 @@ def test_blob_setter(device):
     request.infer()
     res_2 = np.sort(request.output_blobs['fc_out'].buffer)
     assert np.allclose(res_1, res_2, atol=1e-2, rtol=1e-2)
+
+
+def test_blob_setter_with_preprocess(device):
+    ie_core = ie.IECore()
+    net = ie_core.read_network(test_net_xml, test_net_bin)
+    net.input_info['data'].layout = "NHWC"
```

Comment:
I think changing layout here can make harder potential issues localization (I think better separate layout setting changes and preprocessing), BTW below you pass data in NCHW (according to transpose), it can mislead users.

Reply:
fixed